### DISCUSSION TITLE
https://blog.xyx-blog.xyz/%E7%AC%94%E8%AE%B0/%E9%98%BF%E5%AE%BE%E3%81%AE%E7%AC%94%E8%AE%B0/Leon_Bin's%20Blog.html

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#10] [Announcements]
# https://blog.xyx-blog.xyz/%E7%AC%94%E8%AE%B0/%E9%98%BF%E5%AE%BE%E3%81%AE%E7%AC%94%E8%AE%B0/Leon_Bin's%20Blog.html

记录回忆，知识和畅想的地方

https://blog.xyx-blog.xyz/%E7%AC%94%E8%AE%B0/%E9%98%BF%E5%AE%BE%E3%81%AE%E7%AC%94%E8%AE%B0/Leon_Bin's%20Blog.html

**Comment by XYXMichael** [comment]
so cool!!!

**Reply by Leon-Bin** [reply]
😘
